CONTRIBUTING.md

```diff
@@ -67,8 +67,12 @@ public extension SomeType {
 	- Properties & static properties
 	- Methods & static methods
 	- Initializers
-- Files are named based on the type that the contained extensions extend (example: all String extensions are found in "**StringExtensions.swift**" file)
-- extensions are ordered inside files in the following order:
+- Files are named based on the type that the contained extensions extend. (example: all String extensions are found in "**StringExtensions.swift**" file)
+- All extensions should be tested.
+- All extensions files and test files have a one to one relation.(example: all tests for "**StringExtensions.swift**" are found in the "**StringExtensionsTests.swift**" file)
+- All extensions have a one to one relation to a test which should cover all edge cases and is named like the extension. (example: `DateExtensions` method `isBetween` is tested in file: `DateExtensionsTest` test: `testIsBetween`)
```

Comment:
Can we split this into two points?
- There should be a one to one relationship between extensions and their backing tests.
- Tests should be named using the same API of the extension it backs.
(example: `DateExtensions` method `isBetween` is named `testIsBetween`)